hyper/compat.py

```diff
@@ -31,6 +38,9 @@ def ignore_missing():
 if is_py2:
     if is_py2_7_9_or_later:
         import ssl
+    elif is_appengine:
+        from . import ssl_compat_appengine
+        ssl = ssl_compat_appengine
```

Comment:
Is this order of fallbacks correct? It seems like if GAE identifies itself as Python 2.7.9 or later then we won't use the AppEngine compatibility module. Having not used GAE myself I'm not sure if this order is right...